news/2022-05-18-new-featured-artist-joyless.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "New Featured Artist: JOYLESS"
+date: 2022-05-18 09:00:00 +0000
+---
+
+Delivering California surf-punk from the distant shores of Norway, **JOYLESS** joins our Featured Artist collection!
+
+![](https://assets.ppy.sh/artists/267/header.jpg)
+
+osu! is one of the few rhythm games that promotes *every* genre of music, and **JOYLESS** is here to help prove it. These catchy punk tunes are far from average rhythm game content, but mappers who love artists like [*PUP*](https://osu.ppy.sh/beatmaps/artists/79) will discover that **JOYLESS** is their next addiction.
```

Comment:
```suggestion
osu! is one of the few rhythm games that promote *every* genre of music, and **JOYLESS** is here to help prove it. These catchy punk tunes are far from average rhythm game content, but mappers who love artists like [PUP](https://osu.ppy.sh/beatmaps/artists/79) will discover that **JOYLESS** is their next addiction.
```

Reply:
mentions of other artists elsewhere have been written in italics iirc :

![image](https://user-images.githubusercontent.com/36564236/169030183-dedd5562-9325-44e0-9b35-e2a99b1457fd.png)

![image](https://user-images.githubusercontent.com/36564236/169030720-1b1c3366-b187-41ea-9368-d7547bcf677d.png)

Reply:
from what i've observed, it's usually works, games or songs that get italicised, not artists (or mappers, or other types of creators). if this format is preferred for news posts then alright, but i'd rather draw the line consistently

Reply:
yeah i don't think there's been an "established standard" or anything on this just yet

Reply:
it is standardised [on the wiki](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#emphasising) at least. that said, italics can also be used for particular emphasis when reading, independent of formatting, which i'm fine with

Reply:
also "promotes" seems more correct here.

Reply:
promotes definitely reads better to me here out loud at least

Reply:
"osu! is one of the few rhythm games, that promotes"

sure if you parse it like that (promotes referring to "osu!") then it would be sort of correct, although then "the few rhythm games" would be cut short.

compare this to "osu! is one of those few rhythm games that promote..."

would you still read it the same?

Reply:
I think it's more correct as it is, regardless of how you group things.

Reply:
one makes the verb refer to a singular entity and one to a plural one, so no i disagree. (here's a [guide on this matter](https://www.quickanddirtytips.com/education/grammar/singular-or-plural-verbs-with-one-of-the) that discusses it further)